pages/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './navbar.module.scss';
+
+const Navbar = () => {
+  const GITHUB_LOGO = "/icons/github.png"
+  const DEFAULT_AVATAR = '/images/Avatar.png';
+  const [toggle, setToggle] = useState(false);
+  const sidebarToggle = () => {
+    setToggle(!toggle);
+  };
+
+  return (
+    <div className={styles.wrapper}>
+        <nav className={styles.navBar}>
+          <div className={styles.hamburger} onClick={sidebarToggle}>
+            <span className={styles.bar}></span>
+            <span className={styles.bar}></span>
+            <span className={styles.bar}></span>
+          </div>
+          <div className={styles.navBarLogin}>
+            <Link
+              className={styles.btnLogin}
+              href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+            >
+              <button className={styles.btnLoginText} >
+                Sign In
+                <Image
+                  className={styles.githubLogo}
+                  src={GITHUB_LOGO}
+                  alt="GitHub Icon"
+                  height={15}
+                  width={15}
+                />
+              </button>
+            </Link>
+          </div>
+          <ul
+            className={`${styles.navBarMenu} ${toggle ? `${styles.active}` : ''
+              }`}
+          >
+            <li className={styles.navBarLogoLi} >
+              <Link href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97">

```

Comment:
add this link to a constant also why are we shipping this `client ID` ?

Reply:
I removed from code...client id href for now

Reply:
from sign-in(mobile) and sign-in with github(desktop)